### PR TITLE
mailing: change from atom to rss feed

### PR DESCRIPTION
**Pull request by harunurhan**
For weekly jobs feed.

### CODE REVIEW

File: backend/inspirehep/mailing/views.py [hunk 1]
```diff
@@ -56,20 +57,20 @@ def get_weekly_jobs_rss():
...
-        title=title,
-        content=content,
-        content_type="html",
-        author="inspirehep.net",
-        updated=date,
-        published=date,
-    )
-
-    return feed.get_response()
+    date = datetime.utcfromtimestamp(timestamp).replace(tzinfo=pytz.UTC)
```
Comment (pazembrz) [inline backend/inspirehep/mailing/views.py]: why? You have already UTC there.

Comment (harunurhan) [reply backend/inspirehep/mailing/views.py]: nop it doesn't have `tzinfo` and `feedgen` is complaining because of that.

Comment (pazembrz) [reply backend/inspirehep/mailing/views.py]: O. You are right. https://docs.python.org/3.6/library/datetime.html#datetime.datetime.utcfromtimestamp This is weird. 
Cannot you use datetime.fromtimestamp(timestamp, timezone.utc) then?

Comment (harunurhan) [reply backend/inspirehep/mailing/views.py]: yep, that's same result and seems cleaner.

**Pull request merged**